site/content/contribute/webapp/developer-setup.md

```diff
@@ -43,3 +43,7 @@ Set up your development environment for building, running, and testing the Matte
     cd $(go env GOPATH)/src/github.com/mattermost/mattermost-webapp
     make test
     ```
+    
+8. If you want to enable live-reload functionality in the webapp:
```

Comment:
Is there a situation where we would recommend live-reload functionality to be enabled?

Reply:
I'd recommend it for anyone working on the webapp. It's only active under dev mode and will refresh the webapp as you edit the source code, it's rather useful

Reply:
@reflog That's awesome! I'd just propose a couple of updates by including the benefit in the step, and formatting the command similar to previous steps:

```
8. (Optional) Enable live reload functionality to refresh the webapp as you edit the source code. First, install and enable live reload script injection extension for your web browser ([Chrome](https://chrome.google.com/webstore/detail/remotelivereload/jlppknnillhjgiengoigajegdpieppei/related?hl=en) | [Firefox](https://addons.mozilla.org/en-US/firefox/addon/livereload-web-extension/)), then run:

    ```sh
    export MM_LIVE_RELOAD=true
    ```

```

Reply:
@jasonblais - good suggestions! Added.